test/bin/download_images.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+mkdir -p "${VMDISKDIR}"
+
+download_image() {
+    local buildid="${1}"
+    # FIXME: These logs should go to the artifacts directory instead
+    # of the build web server.
+    sudo composer-cli compose logs "${buildid}"
+    sudo composer-cli compose metadata "${buildid}"
+    sudo composer-cli compose image "${buildid}"
+    sudo chown "$(whoami)." ${buildid}-*
+}
+
+echo "Downloading images and metadata to ${IMAGEDIR}"
+pushd "${IMAGEDIR}"
+for blueprint_build in builds/*.edge-commit; do
+    blueprint=$(basename ${blueprint_build} .edge-commit)
+    buildid=$(cat ${blueprint_build})
+    download_image "${buildid}"
+    commit_file="${buildid}-commit.tar"
+    echo "Unpacking ${commit_file} ${blueprint}"
+    tar -C "${IMAGEDIR}" -xf "${commit_file}"
+done
+
+echo "Updating references"
+(cd "${IMAGEDIR}" && ostree --repo=repo refs)
```

Comment:
`ostree refs` only lists the references, `ostree summary -u` is required instead
are you sure we can just extract all commits into one directory without using `ostree pull-local`?